playground/src/components/module_requests/request_submission_selection.tsx

```diff
@@ -99,10 +104,10 @@ export default function SelectSubmission({
   } = useSubmissions(mode, exercise);
 
   const responseSubmissionView = (response: ModuleResponse | undefined) => {
-    if (!response || response.status !== 200 || typeof response.data !== "number") {
+    if (!response || response.status !== 200 || typeof response.data.data !== "number") {
```

Comment:
```suggestion
    if (!response || response.status !== 200 || typeof response.data.data !== "number") {
```
It looks like a code smell to me that you must write `.data.data` here.

Reply:
fixed